src/main/java/org/sagebionetworks/web/client/jsinterop/ReactComponentProps.java

```diff
@@ -18,29 +17,6 @@ public interface CallbackRef {
     void run(Element element);
   }
 
-  public JsArray<ReactElement<?>> children;
-
```

Comment:
This `children` prop is unused after these changes. Children will be appended as arguments to `React.createElement`